setup.cfg

```diff
@@ -1,6 +1,21 @@
+[bdist_wheel]
+universal=1
+
 [metadata]
+license_file = LICENSE.txt
 description-file = README.markdown
 
+[check-manifest]
+ignore =
+  bin/*
+  dev_tool/*
+  test/fixture/cli/vital.vim/*
+  CONTRIBUTING.rst
+  VERSION.txt
```

Comment:
is this file used anywhere? 

It contains `0.4a1`

But when an sdist is created, the tarball version is `vim-vint-0.3.21.dev38+g4c3c11f` so something is a bit off

Reply:
It was forgotten when switching to setuptools-scm.
Will remove it in https://github.com/Kuniwak/vint/pull/325.

The version via setuptools-scm is off, since it takes the merged release branch.
Fixed in https://github.com/pypa/setuptools_scm/pull/346.

We could work around it already, but I plan to use setupmeta instead anyway (#323).

Long story short, `VERSION.txt` is not there anymore.

Reply:
Should be ready now.